python/cugraph-pyg/cugraph_pyg/tests/mg/test_mg_cugraph_store.py

```diff
@@ -184,7 +182,7 @@ def test_renumber_vertices_basic(single_vertex_graph, dask_client):
     )
 
     index = cugraph_store._get_vertex_groups_from_sample(nodes_of_interest)
-    assert index["vt1"].tolist() == sorted(nodes_of_interest.tolist())
+    assert index["vt1"].tolist() == nodes_of_interest.tolist()
```

Comment:
No suggestion here, but I'm curious why `sorted()` was removed (or why it wasn't added to both lists being compared), since wouldn't that be a more reliable compare? Or, does order matter?

Reply:
Order does matter now, that is one of the changes in this PR.  Before, the nodes of interest were sorted, but that is not correct - they should be returned in order of appearance.